library/VTKExtensions/Rendering/vtkF3DRendererWithColoring.cxx

```diff
@@ -114,17 +108,22 @@ void vtkF3DRendererWithColoring::SetUseInverseOpacityFunction(bool use)
   if (this->UseInverseOpacityFunction != use)
   {
     this->UseInverseOpacityFunction = use;
-    if (this->VolumeProp)
+
+    auto volPropsAndMappers = this->Importer->GetVolumePropsAndMappers();
```

Comment:
you probably want a reference here: `auto&`. Const-ness is automatically deduced so you don't have to specify it.

Reply:
const is needed though.